src/presentation/http/router/note.test.ts

```diff
@@ -243,38 +244,7 @@ describe('Note API', () => {
   });
 
   describe('PATCH note/:notePublicId ', () => {
-    test.each([
-      /** Returns 200 if user is team member with a Write role */
-      {
-        role: MemberRole.Write,
-        isAuthorized: true,
-        expectedStatusCode: 200,
-      },
-
-      /** Returns 403 if user is team member with a Read role */
-      {
-        role: MemberRole.Read,
-        isAuthorized: true,
-        expectedStatusCode: 403,
-        expectedMessage: 'Permission denied',
-      },
-
-      /** Returns 403 if user is not in the team */
-      {
-        role: null,
-        isAuthorized: true,
-        expectedStatusCode: 403,
-        expectedMessage: 'Permission denied',
-      },
-
-      /** Returns 401 if user is not authorized */
-      {
-        role: null,
-        isAuthorized: false,
-        expectedStatusCode: 401,
-        expectedMessage: 'You must be authenticated to access this resource',
-      },
-    ])
+    test.each(memberRightsTestData)
```

Comment:
maybe rename `memberRightsTestData` to `accessRight`? So that it would read naturally like "test each access right"